src/main/java/com/grud/Grud.java

```diff
@@ -0,0 +1,64 @@
+package com.grud;
+
+import lombok.SneakyThrows;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringJoiner;
+
+public class Grud {
```

Comment:
It is better to create a crud repository per table.

Reply:
please explain more

Reply:
Separate crud repository for company, customer, developers,....